applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -442,19 +426,18 @@ void RegisterNodeSurfaceGeometry(
     pybind11::module& m,
     const std::string& name)
 {
-    namespace py = pybind11;
     using namespace pybind11::literals;
 
     using VectorType = Kratos::array_1d<double, 3>;
 
     using Type = NodeSurfaceGeometry3D;
-    using Pointer = ANurbs::Pointer<Type>;
+    using Holder = shared_ptr<Type>;
     using Base = ANurbs::SurfaceGeometryBase<VectorType>;
 
     using VariableComponent = Kratos::VariableComponent<
         Kratos::VectorComponentAdaptor<Kratos::array_1d<double, 3>>>;
 
-    pybind11::class_<Type, Base, Pointer>(m, name.c_str())
+    pybind11::class_<Type, Base, Holder>(m, name.c_str())
```

Comment:
this is easy to read and hard to read at the same time :D
Fine for me

Reply:
`Holder` is from the pybind11 documentation